Sources/XMTP/Messages/Topic.swift

```diff
@@ -30,6 +31,8 @@ public enum Topic {
 			return wrap("dm-\(addresses)")
 		case let .directMessageV2(randomString):
 			return wrap("m-\(randomString)")
+		case let .allowList(identifier):
+			return wrap("privatestore-\(identifier)/allowlist")
```

Comment:
@neekolas @rygine what is that we want this to be called? just `pppp-\(identifier)`

Reply:
my vote is for `pppp-\(identifier)/allowlist`

Reply:
Per conversation in JS we're going to go with `pppp-\(identifier)` to help keep the topics from getting too general.